apps/mobile/src/components/TradeChecklistFlow/atoms/updatesToBeSentAtom.ts

```diff
@@ -29,12 +29,20 @@ const updatesToBeSentAtom = atom<TradeChecklistUpdate>(
 )
 export default updatesToBeSentAtom
 
+export const areThereUpdatesToBeSentAtom = atom(
+  (get) => get(updatesToBeSentAtom) !== UPDATES_TO_BE_SENT_INITIAL_STATE
```

Comment:
Can we check this with deep equal? I don't like comparing direct objects like that. I understand that `clearUpdateToBeSendActionAtom` always set this exact instance, but the atom accepts any `{}` instance you give it.

It might make sense, to test it even a bit more thoroughly like: `Object.keys(updatesToBeSentAtom).some(key => !!updatesToBeSentAtom[key])`, typecheck will pass even if you set allowed keys to `undefined`